build.gradle

```diff
@@ -355,6 +355,8 @@ dependencies {
     compile 'org.broadinstitute:gatk-bwamem-jni:1.0.4'
     compile 'org.broadinstitute:gatk-fermilite-jni:1.2.0'
 
+    compile 'org.broadinstitute:http-nio:0.1.0-rc1'
```

Comment:
Confirming that this artifact is now in maven central?

Reply:
If the tests ran it's in maven central.  Anything that doesn't have "SNAPSHOT" on the end goes there.